### PR TITLE
adding dreambooth example

### PR DESCRIPTION
**Pull request by benartuso**

### CODE REVIEW

File: examples/dreambooth_custom_character/README.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+2. If you train or fine-tune open source models with your own data rather than remaining beholden to third-party services, you can embed the model directly into your application. This will likely save you money over purchasing credits to existing providers, and eliminate your dependency on a vendor that could change its API / pricing or stop operating at any time.
+3. Fine-tuning with your own proprietary data allows you to gain a competitive advantage that you can’t access with prompting alone. If your IP relies solely on the specific way you prompt a publicly accessible model, it’s much easier for competitors to replicate your results!
+
+At [Oxen](https://oxen.ai/), we’re firm believers in the power of enhancing foundational models with domain-specific data. In this tutorial, we’ll use an Oxen repository and the [Dreambooth](https://dreambooth.github.io/) fine-tuning technique to create a Stable Diffusion variant that can consistently reproduce cute, fluffy cartoon oxen like the one currently hanging out [on our 404 page](https://www.oxen.ai/not-a-real-url).
```
Comment (gschoeni) [inline examples/dreambooth_custom_character/README.md]: I think we should move this above the 1,2,3 reasons why you should fine tune, just to get to the punchline of what we are about to do quicker.

File: examples/dreambooth_custom_character/README.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
+# Creating a Cute Custom Character with Stable Diffusion and Dreambooth
+
+## Introduction
+[Stable Diffusion](https://github.com/CompVis/stable-diffusion) is an incredible open-source tool for fast, effective generation of novel images across a wide variety of domains. Despite its power and convenience, it can struggle to create consistent, identifiable characters in specific styles.
+
+While improvements in prompting can mitigate this to some extent, it is almost always a better bet to fine-tune the model with your own data. Here are a few reasons why:
```
Comment (gschoeni) [inline examples/dreambooth_custom_character/README.md]: Let's move this below our 404 ox, and add a title "Why fine-tune in the first place?" or something

Comment (gschoeni) [reply examples/dreambooth_custom_character/README.md]: Then that leads well into our "quick preview with only 6 images"

File: examples/dreambooth_custom_character/README.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+**Before Dreambooth:**
+![4 generated images under the prompt "an image of a happy white fluffy cartoon ox eating almonds", showing limited coherence / consistency](images/happy-fluffy-before.png)
+![4 generated images under the prompt "the oxenai ox eating almonds", showing limited coherence / consistency](images/oxenai-ox-before.png)
+
+**After Dreambooth:**
+![4 generated images under the same "fluffy cartoon ox" prompt as above, but more stylistically coherent](images/happy-fluffy-after.png)
+![4 generated images under the same "oxenai ox" prompt as above, but more stylistically coherent](images/oxenai-ox-after.png)
+
+These are some pretty exciting results, both in the increased coherence and consistency of the generated images, and the ease with which we could teach the model to recognize "the oxenai ox" as a specific token. Let's get started!
```
Comment (gschoeni) [inline examples/dreambooth_custom_character/README.md:28]: I think we need to introduce the concept of a specific token you are fine tuning the model with before we mention the "oxenai ox". Just a quick title and blurb on how fine-tuning a token works and why we would do it.

File: examples/dreambooth_custom_character/README.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+  --lr_scheduler="constant" \
+  --lr_warmup_steps=0 \
+  --max_train_steps=400 \
+  --push_to_hub
+```
+
+This took around 5 minutes to run on a single NVIDIA A10 with 24GB of GPU VRAM, which we rented from [Lambda Labs](https://lambdalabs.com/).
+
+### Generating new images 
+If the `--push_to_hub` flag was set, the script will have pushed the resulting model up to Hugging Face after training. As such, we can modify our inference UI code as following to run inference on the new model:
```
Comment (gschoeni) [inline examples/dreambooth_custom_character/README.md:160]: Eventually.....I think we should be blog storage for models as well. But this is fine for now.

File: examples/dreambooth_custom_character/README.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+
+As an attempt to solve both these issues, we’re going to use this new model (and an [Oxen-powered RLHF interface](https://blog.oxen.ai/collecting-data-from-human-feedback-for-generative-ai/)) to gain further, prompt-specific feedback for additional fine-tuning. We think that this will be key in bridging the gap to a generalizable, consistent character—stay tuned for the results! 
+
+At OxenAI we want to see what you are building! Reach out at hello@oxen.ai, follow us on Twitter [@oxendrove](https://twitter.com/oxendrove), dive deeper into the [documentation](https://github.com/Oxen-AI/oxen-release), or **Sign up for Oxen today. http://oxen.ai/register.**
+
+And remember—for every star on [GitHub](https://github.com/Oxen-AI/oxen-release), an ox gets its wings.
+
+[No, really...go check it out!](https://oxen.ai/ox/FlyingOxen) 
+
+![An AI-generated image of a purple, magestic, winged space ox](images/winged-ox.png)
```
Comment (gschoeni) [inline examples/dreambooth_custom_character/README.md]: can we choose a slightly more wing-y ox lol this one looks like the wings aren't attached to it's body

File: examples/dreambooth_custom_character/README.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+
+Additionally, it’s nailed the general characteristics of our desired ox (cute, cartoonish, fuzzy) and is showing much-improved consistency, but isn’t yet able to produce a singular, recognizable character. 
+
+As an attempt to solve both these issues, we’re going to use this new model (and an [Oxen-powered RLHF interface](https://blog.oxen.ai/collecting-data-from-human-feedback-for-generative-ai/)) to gain further, prompt-specific feedback for additional fine-tuning. We think that this will be key in bridging the gap to a generalizable, consistent character—stay tuned for the results! 
+
+At OxenAI we want to see what you are building! Reach out at hello@oxen.ai, follow us on Twitter [@oxendrove](https://twitter.com/oxendrove), dive deeper into the [documentation](https://github.com/Oxen-AI/oxen-release), or **Sign up for Oxen today. http://oxen.ai/register.**
+
+And remember—for every star on [GitHub](https://github.com/Oxen-AI/oxen-release), an ox gets its wings.
+
+[No, really...go check it out!](https://oxen.ai/ox/FlyingOxen)
```
Comment (gschoeni) [inline examples/dreambooth_custom_character/README.md]: No, really... We hooked up an Oxen Repo to a GitHub web-hook that runs Stable Diffusion every time we get a star. Go find yours!

File: examples/dreambooth_custom_character/README.md (hunk 4 above)
Comment (benartuso) [reply examples/dreambooth_custom_character/README.md:160]: agreed

**Comment by gschoeni** [comment]
Looking good to me! When you release the blog can you also add instructions somewhere for how you deploy to medium and ghost? Maybe our Notion page?

**Pull request merged**